src/RouteResolver.tsx

```diff
@@ -73,9 +76,14 @@ export const generateSynapseObject = (
   const { props, ...rest } = synapseConfig
   const key = JSON.stringify(props)
   return (
-    <TokenContext.Consumer key={key}>
-      {(value: string) => {
-        const propsWithSearchAndToken = { ...props, searchParams, token: value }
+    <SynapseContextConsumer key={key}>
+      {(ctx?: SynapseContextType) => {
+        const propsWithSearchAndToken = {
+          ...props,
+          searchParams,
+          token: ctx?.accessToken,
+          accessToken: ctx?.accessToken,
```

Comment:
Some components (in both SRC and portals) still take `token` or `accessToken` as a prop, so we cannot remove this yet.